couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -45,23 +45,29 @@ def __init__(self):
     def __init__(self, name, init_config, instances):
         super(Couchbase, self).__init__(name, init_config, instances)
 
+        self._server = self.instance.get('server', None)
+        if self._server is None:
+            raise ConfigurationError("The server must be specified")
+        self._tags = list(set(self.instance.get('tags', [])))
+        self._tags.append('instance:{}'.format(self._server))
+
         # Keep track of all instances
         self._instance_states = defaultdict(lambda: self.CouchbaseInstanceState())
 
-    def _create_metrics(self, data, instance_state, server, tags=None):
+    def _create_metrics(self, data):
         # Get storage metrics
         storage_totals = data['stats']['storageTotals']
         for key, storage_type in storage_totals.items():
             for metric_name, val in storage_type.items():
                 if val is not None:
                     metric_name = 'couchbase.{}.{}'.format(key, self.camel_case_to_joined_lower(metric_name))
-                    self.gauge(metric_name, val, tags=tags)
+                    self.gauge(metric_name, val, tags=self._tags)
 
         # Get bucket metrics
         for bucket_name, bucket_stats in data['buckets'].items():
             metric_tags = ['bucket:{}'.format(bucket_name), 'device:{}'.format(bucket_name)]
-            if tags:
-                metric_tags.extend(tags)
+            if self._tags:
+                metric_tags.extend(self._tags)
```

Comment:
`self._tags` should always evaluate to something - so can avoid the `if` clause here